### PR TITLE
Clean up test for /user views

### PR DESCRIPTION
**Pull request by Russell-Waterhouse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +89,7 @@ This will take all the migrations that haven't been applied and runs them agains
 For development purposes, The PostgreSQL can be connected to via its Docker container shell, to view database contents or metadata. To get a bash shell, run:
 
 ```
-docker exec -it back-end_db_1 /bin/bash
+docker exec -it back-end-db-1 /bin/bash
```
Comment (skosman) [inline README.md:92]: On Mac, this command works with the old underscore name, not the updated one. Is the container name different for linux? If so, should we add a comment for both cases

**Review by skosman — APPROVED** [review]
Other than one small comment, it looks good! Thanks for updating the tests to use APIClient

File: README.md (hunk 1 above)
Comment (Russell-Waterhouse) [reply README.md:92]: Oh yeah it's different on linux, I'll add a comment I just assumed it was the same everywhere

Comment (skosman) [reply README.md:98]: Thanks Russ!

**Pull request merged**